oolong-core/src/main/scala/ru/tinkoff/oolong/QueryMeta.scala

```diff
@@ -0,0 +1,194 @@
+package ru.tinkoff.oolong
+
+import java.util.regex.Pattern
+import scala.quoted.*
+
+import ru.tinkoff.oolong.Utils.*
+
+case class QueryMeta[T](map: Map[String, String])
+
+extension [T](inline q: QueryMeta[T])
+  inline def withRenaming(inline fields: (T => (Any, String))*): QueryMeta[T] = ${
```

Comment:
We rejected similar interface for update because it was harder to implement such one there
Here it was implemented and though it brings a little overhead in compile-time, I think 
```QueryMeta. withCamelCase.withRenaming(_.field -> "new_name)``` 
is better and more beautiful syntax